modules/client-configuration/pages/supported-features-ubuntu.adoc

```diff
@@ -19,58 +19,176 @@ The operating system you run on a client is supported by the organization that s
+* {cross} the feature is not available
+* {question} the feature is under consideration, and may or may not be made available at a later date
+* ``{trad}`` the feature is supported only on traditional clients
+* ``{salt}`` the feature is supported only on Salt clients.
 
 
 [cols="1,1,1", options="header"]
 .Supported Features on {ubuntu} Operating Systems
 |===
-| Feature | {ubuntu}{nbsp}16.04 | {ubuntu}{nbsp}18.04
-| Client | icon:check[role="green"] | icon:check[role="green"]
-| System packages | Canonical | Canonical
-| Registration | ``Salt`` | ``Salt``
-| Install packages | ``Salt`` | ``Salt``
-| Apply patches | icon:question[role="gray"] | icon:question[role="gray"]
-| Remote commands | ``Salt`` | ``Salt``
-| System package states | ``Salt`` | ``Salt``
-| System custom states | ``Salt`` | ``Salt``
-| Group custom states | ``Salt`` | ``Salt``
-| Organization custom states    | ``Salt`` | ``Salt``
-| System set manager (SSM) | ``Salt`` | ``Salt``
-| Service pack migration | N/A | N/A
-| Basic Virtual Guest Management icon:asterisk[role="none"] | ``Salt`` | ``Salt``
-| Advanced Virtual Guest Management icon:asterisk[role="none"] | ``Salt`` | ``Salt``
-| Virtual Guest Installation (Kickstart), as Host OS | icon:times[role="danger"] | icon:times[role="danger"]
-| Virtual Guest Installation (image template), as Host OS | ``Salt`` | ``Salt``
-| System deployment (PXE/Kickstart) | icon:times[role="danger"] | icon:times[role="danger"]
-| System redeployment (Kickstart) | icon:times[role="danger"] | icon:times[role="danger"]
-| Contact methods | ``Salt``: ZeroMQ, Salt-SSH | ``Salt``: ZeroMQ, Salt-SSH
-| Works with {productname} Proxy | ``Salt`` | ``Salt``
-| Action chains | ``Salt`` | ``Salt``
-| Software crash reporting | icon:times[role="danger"] | icon:times[role="danger"]
-| Staging (pre-download of packages) |  ``Salt`` | ``Salt``
-| Duplicate package reporting | ``Salt`` | ``Salt``
-| CVE auditing |  icon:question[role="gray"] | icon:question[role="gray"]
-| SCAP auditing | icon:question[role="gray"] | icon:question[role="gray"]
-| Package verification | icon:times[role="danger"] | icon:times[role="danger"]
-| Package locking |  ``Salt`` | icon:times[role="danger"]
-| System locking |   ``Salt`` | icon:times[role="danger"]
-| System snapshot | icon:times[role="danger"] | icon:times[role="danger"]
-| Configuration file management |  ``Salt`` | ``Salt``
-| Package profiles |  ``Salt``: Profiles supported, Sync not supported | ``Salt``: Profiles supported, Sync not supported
-| Power management |  icon:check[role="green"] | icon:check[role="green"]
-| Monitoring |  icon:times[role="danger"]  | ``Salt``
-| Docker buildhost |  icon:question[role="gray"]   | icon:question[role="gray"]
-| Build Docker image with OS | ``Salt`` | ``Salt``
-| Kiwi buildhost |   icon:times[role="danger"] | icon:times[role="danger"]
-| Build Kiwi image with OS |  icon:times[role="danger"] | icon:times[role="danger"]
+
+| Feature
```

Comment:
This way of formatting the tables takes up more lines, but makes the source much easier to read. It should make it easier on our reviewers too (@juliogonzalez ?).

Reply:
Also, I don't really know why we have markup on the "Salt" and "Traditional" entities. Removing it might be sensible.